dali/python/backend_impl.cc

```diff
@@ -1792,7 +1792,10 @@ PYBIND11_MODULE(backend_impl, m) {
     .export_values();
 
   // DALIInterpType
-  py::enum_<DALIInterpType>(types_m, "DALIInterpType", "Interpolation mode\n<SPHINX_IGNORE>")
+  py::enum_<DALIInterpType>(types_m, "DALIInterpType", "Interpolation mode.\n"
+          "Linear and Cubic interpolations in DALI are separable with regards to dimensions. "
+          "Therefore, ``DALI_INTERP_LINEAR`` and ``DALI_INTERP_CUBIC`` modes capture all: "
+          "Linear, Bi-linear, Tri-linear etc... interpolation modes.\n<SPHINX_IGNORE>")
```

Comment:
If Sphinx can't read per-value docstrings, I'd simplify the message:
```suggestion
  py::enum_<DALIInterpType>(types_m, "DALIInterpType", "Interpolation mode.\n"
          "Note: for 2D inputs, linear and cubic are synonymous with bilinear and bicubic, respectively.")
```

Reply:
Changed this too